### ISSUE TITLE
Regulator augment not updating when entering the value through the keyboard.

### ISSUE THREAD

**Issue by Cerbrus** [desht/ModularRouters#114]
### Minecraft Version
1.16

### Forge Version
1.16.4-forge-35.1.36

I'm using the "FTB Presents Direwolf20 1.16" modpack, version 1.4.1

### Mod Version
modular-routers-1.16.4-7.3.0-47

### Describe your problem, including steps to reproduce it
The regulator augment, when inserted in a player module, doesn't update the value when you type in the input field.  
Only when you scroll in the input field, you'll see the augment's item count update, on the player module.

**Comment by desht** [comment]
Sorry, not seeing that behaviour.  Typing in the augment's textfield works just fine for me.

**Comment by Jimmeh42** [comment]
I have had this issue with the FTB Direwolf20 pack for 1.16 it must be a conflict with another mod causing it.
Using the scroll wheel while hovering over the augment slot works correctly.

**Comment by desht** [comment]
Hmm, do you see a warning like the following in the server log?

```
[14:49:35] [Server thread/WARN] [me.de.mo.ModularRouters/]: ignoring ModuleSettingsMessage for desht - expected module not found @ me.desht.modularrouters.util.MFLocator@40b1c8af
```

**Comment by desht** [comment]
Yeah, found the problem.  Will be fixed in 7.4.0.

Keypresses were being passed through the GUI and not swallowed by the textfield, causing the held item to be changed.  So when the client sent the update packet, the held item was no longer what the server was expecting, so it ignored the update.

**Comment by desht** [comment]
Fixed in 7.4.0
